OnionGalaxy/TwinlightSouls
# Feature/ui dialogues

What's new:

* Dialogue System. 
* Camera improvements for multiplayer.

HOW TO TEST:
Press U in order to start the dialogues. Then pass it with the SpaceBar or with the lower button (A in Xbox Controller, and X in PlayStation Controller) and then see the results.

It's just the system, I left it for the UI and artist to make it beauty.
